frontend/server.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express')
+const serveStatic = require('serve-static')
+const path = require('path')
+
+const app = express()
+
+app.disable("x-powered-by");
```

Comment:
Great addition for security!

Reply:
Thanks. Actually, SonarCloud gave me this recommendation.